vars/hadoop_vars.yml.template

```diff
@@ -4,4 +4,5 @@ hadoop_usr_dir: "/usr/lib/hadoop"   #this is the symlink to the extracted/instal
 hadoop_lib_dir: "/var/lib/hadoop"
 hadoop_log_dir: "/var/log/hadoop"
 hadoop_run_dir: "/run/hadoop"
+hadoop_http_user: "emma"
```

Comment:
When we connect with hadoop web-ui, the default user is dr-who. I decided to use emma, since it is also the user used for the JupyterNotebook. It does not need to be Unix user. 

A delete action performed through the web-ui would required write permission for dr-who. Like this it is for emma, also the user used for Jupyter notebook. 

Would you like a different user?